src/lib/metricsMgr/MetricsManager.cpp

```diff
@@ -22,30 +22,114 @@
 *
 * Author: Ken Zangelin
 */
-
+#include <sys/time.h>
```

Comment:
Must be kept:

    void MetricsManager::semTake(void)
    {
    if (semWaitStatistics)
    {
      struct timeval start;
      struct timeval end;
      
      gettimeofday(&start, NULL);
      sem_wait(&sem);
      gettimeofday(&end, NULL);
      ...

NTC